tests/fixtures/db_fixtures.py

```diff
@@ -31,7 +32,7 @@ def auth_token():
             }
         }
     }
-    auth_token = b64encode(json.dumps(identity).encode('utf-8'))
+    auth_token = codecs.decode(b64encode(json.dumps(identity).encode('utf-8')))
```

Comment:
I think we don't need codecs here, I am able to convert bytes to string using,
```suggestion
    auth_token = b64encode(json.dumps(identity).encode('utf-8')).decode()
```

Reply:
+1 Thank you for pointing this out. Corrected at both places.